Py/DICOMTID1500Plugin.py

```diff
@@ -124,24 +124,24 @@ def load(self, loadable):
       segLoadables = segPlugin.examine([slicer.dicomDatabase.filesForSeries(segSeriesInstanceUID)])
       for segLoadable in segLoadables:
         segPlugin.load(segLoadable)
-        if hasattr(segLoadable, "referencedSeriesUID"):
+        if len(loadable.ReferencedRWVMSeriesInstanceUIDs)>0:
           # determine if RWVM needs to be applied to the series referenced from SEG
           loadRWVM = False
-          rwvmUIDs = loadable.ReferencedRWVMSeriesInstanceUIDs
-          rwvmFiles = slicer.dicomDatabase.filesForSeries(rwvmUIDs)
-          if len(rwvmUIDs)>0 and len(rwvmFiles)>0:
+          rwvmUID = loadable.ReferencedRWVMSeriesInstanceUIDs[0]
+          logging.debug("Looking up series %s from database" % rwvmUID)
+          rwvmFiles = slicer.dicomDatabase.filesForSeries(rwvmUID)
+          if len(rwvmFiles)>0:
             # consider only the first item on the list - there should be only
             # one anyway, for the cases we are handling at the moment
             rwvmFile = rwvmFiles[0]
             logging.debug("Reading RWVM from "+rwvmFile)
             rwvmDataset = dicom.read_file(rwvmFile)
-            if hasattr(rwvmDataset,"ReferencedSeriesSequence"):              
+            if hasattr(rwvmDataset,"ReferencedSeriesSequence"):
               if hasattr(rwvmDataset.ReferencedSeriesSequence[0],"SeriesInstanceUID"):
                 if rwvmDataset.ReferencedSeriesSequence[0].SeriesInstanceUID == segLoadable.referencedSeriesUID:
```

Comment:
Can we be sure that segLoadable.referencedSeriesUID exists? You removed the check from line 127